Cue/app/tabs/search/SearchListView.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { Navigator, View, Text, ListView, Image, Platform, Dimensions } from 'react-native'
+
+import { connect } from 'react-redux'
+
+import type { DeckMetaData, Card } from '../../api/types'
+
+import EmptyView from '../../common/EmptyView'
+import CueIcons from '../../common/CueIcons'
+
+import SearchListViewItem from './SearchListViewItem'

```

Comment:
This should be `SearchListView`, though I would prefer `SearchResultsListView` to be more explicit.